_rules/aria-allowed-child-element-with-presentational-parent-1g88p9.md

```diff
@@ -0,0 +1,171 @@
+---
+id: 1g88p9
+name: ARIA allowed child element of another element with presentational role
+rule_type: atomic
+description: |
+  This rule checks that allowed child element of another element with presentational role does not cause presentational roles conflicts
```

Comment:
This rule seems like a subset of our `ARIA required context role` rule. It requires that elements that can't exist without a parent have that parent. The rule you propose is more focused, but it doesn't seem to cover anything that isn't also covered by the `required context` rule. Or at least, I can't find anything that isn't.

Would a better solution be to add the one or both of the failed examples to `required context` instead?

Reply:
I see the point. 

The intent was to cover a specific ARIA requirement in detail and comprehensively (Presentational Roles Conflict Resolution) to guarantee authors are clear on what to do in these scenarios. 

I'm uncertain about the next steps. 

The other two existing ARIA requirements are rather more vague but do cover the necessary aspects. If we decide to revert these new rules, we'll still need to add examples to the existing rules.